packages/lesmis-server/examples.http

```diff
@@ -41,3 +41,9 @@ Content-Type: {{contentType}}
 GET http://{{host}}/{{version}}/user HTTP/1.1
 content-type: {{contentType}}
 Cookie: {{sessionCookie}}
+
+
+### Logged in reports fetch
+GET http://{{host}}/{{version}}/reportData/LA_Phonhong%20District/Laos_Schools_Number_Of_Schools_By_Type_Table?type=view&dashboardGroupId=337 HTTP/1.1
+content-type: {{contentType}}
+Cookie: {{sessionCookie}}
```

Comment:
Love these! 👍

Reply:
Thanks! Turns out you don't need to explicitly set the Cookie, the api client will take care of that just like a browser when the response from login sets it. Next PR removes them.